packages/status-react/src/routes/chat/index.tsx

```diff
@@ -43,11 +48,28 @@ const Body = () => {
   const chat = client.community.getChat(chatId)!
   const messages = useMessages(chatId)
 
-  const contentRef = useRef<HTMLDivElement>(null)
+  const location = useLocation()
+  const selectedMesssageId = (
+    location.state as { selectedMesssageId: string } | undefined
+  )?.selectedMesssageId
+
+  const contentRef = useRef<HTMLDivElement | null>(null)
+  // todo: more scrolling conditions
```

Comment:
follow-up: discuss: for example, don't scroll to the bottom if the oldest unread message would be off screen